scripts/setup_fvp_optee.sh

```diff
@@ -216,7 +218,8 @@ cat >> $DST_GEN_ROOTFS/filelist-tee.txt << EOF
 # OP-TEE device
 dir /lib/modules 755 0 0
 dir /lib/modules/$KERNEL_VERSION 755 0 0
-file /lib/modules/$KERNEL_VERSION/optee.ko $DST_OPTEE_LK/optee.ko 755 0 0
+file /lib/modules/$KERNEL_VERSION/optee.ko $DST_OPTEE_LK/core/optee.ko 755 0 0
+file /lib/modules/$KERNEL_VERSION/optee_armtz.ko $DST_OPTEE_LK/armtz/optee_armtz.ko 755 0 0
```

Comment:
For avoiding confusion, I would suggest to replace _LK_ or _lk_  by _LINUXDRIVER_ or _linuxdriver_.